code/src/WijmoProvider/Columns/AbstractProviderColumn.ts

```diff
@@ -6,7 +6,7 @@ namespace WijmoProvider.Column {
     export abstract class AbstractProviderColumn<
         T extends OSFramework.Configuration.IConfigurationColumn
     > extends OSFramework.Column.AbstractColumn<T> {
-        private _provider: wijmo.grid.Column;
+        private _provider: wijmo.grid.ColumnGroup;
```

Comment:
All columns are now of ColumnGroup type?..
Sounds confusing... can you put a comment?